packages/ng/styles/definitions/user/user-tile.scss

```diff
@@ -1,92 +1,84 @@
 @mixin userTileStyle {
-	:host {
-		display: flex;
-		align-items: center;
-		padding: 0.5em;
+	display: flex;
+	align-items: center;
+	padding: 0.5em;
 
-		lu-user-picture {
-			margin-right: 0.5em;
-		}
+	lu-user-picture {
+		margin-right: 0.5em;
+	}
 
-		.user-tile-title {
-			font-size: 0.9em;
-			font-weight: 600;
-			line-height: 1.3;
-			margin: 0;
-		}
+	.user-tile-title {
+		font-size: 0.9em;
+		font-weight: 600;
+		line-height: 1.3;
+		margin: 0;
+	}
 
-		.user-tile-label {
-			display: block;
-      font-size: var(--sizes-small-font-size);
-			line-height: 1.1;
-      color: var(--palettes-grey-600);
-		}
+	.user-tile-label {
+		display: block;
+    font-size: var(--sizes-small-font-size);
+		line-height: 1.1;
+    color: var(--palettes-grey-600);
+	}
 
-		.user-tile-footnote {
-			display: block;
-			font-size: var(--sizes-small-font-size);
-			line-height: var(--sizes-small-line-height);
-		}
+	.user-tile-footnote {
+		display: block;
+		font-size: var(--sizes-small-font-size);
+		line-height: var(--sizes-small-line-height);
+	}
 
-		.user-info {
-			margin: auto 0;
-			word-break: break-all;
-		}
+	.user-info {
+		margin: auto 0;
+		word-break: break-all;
```

Comment:
There's this issue : https://github.com/LuccaSA/lucca-front/issues/1489. Maybe we could address (or choose to close) it in this commit ?

Reply:
✅